website_product_document_download_counter/models/product_document.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2025 Cetmix OÜ
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class ProductDocument(models.Model):
+    _inherit = "product.document"
+
+    download_count = fields.Integer(
+        default=0,
+        readonly=True,
+        copy=False,
+        help="Number of times this document has been downloaded",
+    )
+
+    count_downloads_enabled = fields.Boolean(
```

Comment:
My suggestion was for having the same prefix, and avoiding plurals in the middle of the word:

```suggestion
    download_count_enabled = fields.Boolean(
```

See https://www.youtube.com/watch?v=aU9fHG4K6Yo for some reasoning about the plurals.

Reply:
Got it, fixed now.